### PR TITLE
Kevin Editing Scene Metadata On the Scene Modal 

### PR DESCRIPTION
**Pull request by andstun**
Added a scene modal to the "Edit" button to allow for users to make changes the scene name and background_id. Some strange warnings pop up which don't seem to affect functionality but might need to be dealt with in the future (in the meantime I will to try to resolve them myself)

https://www.loom.com/share/c6f44ac08f2d4419bf5e31b2dabe1307

### CODE REVIEW

File: public/Aframe/samples/loading-scene-1/components.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ AFRAME.registerComponent("gravity-switcher", {
             const physics = el.getAttribute("physics");
             let gravity = physics["gravity"];
             gravity *= -1;
+
```
Comment (ahmedhamodi) [inline public/Aframe/samples/loading-scene-1/components.js:117]: Let's undo this change :)

Comment (andstun) [reply public/Aframe/samples/loading-scene-1/components.js:117]: Just noticed that myself lol

File: src/Components/Admin/EnvironmentEditor/environment.js [hunk 2]
```diff
@@ -161,6 +187,9 @@ export default function EnvironmentEditor({
                                                     <SceneCard
                                                         scene={scene}
                                                         index={index}
+                                                        handleEditClick={
+                                                            onCreateButtonClick
+                                                        }
```
Comment (aaronabraham311) [inline src/Components/Admin/EnvironmentEditor/environment.js:192]: Not sure if this is linted

File: src/Components/Admin/EnvironmentEditor/environment.js [hunk 3]
```diff
@@ -57,6 +57,8 @@ export default function EnvironmentEditor({
     const [environment, setEnvironment] = useState({});
     const [scenes, setScenes] = useState([]);
     const [createModalOpen, setCreateModalOpen] = useState(false);
+    const [editModalOpen, setEditModalOpen] = useState(false);
+    const [editSceneCard, setEditSceneCard] = useState({});
```
Comment (andstun) [inline src/Components/Admin/EnvironmentEditor/environment.js:61]: @aaronabraham311 @ahmedhamodi what do you guys think of this state name? I tried to name it something other than "editScene" to avoid a conflict with the "editScene" function in sceneEndpoints.js, but my naming sense is still pretty unrefined

**Review by ahmedhamodi — COMMENTED** [review]
Code changes look good on first pass.

Will test locally tomorrow morning since I'm about to wrap up for the day. Thanks for working on this outside of work sessions Kevin 🎉 !

File: src/Components/Admin/EnvironmentEditor/environment.js (hunk 3 above)
Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/environment.js:61]: I think it's fine!

Comment (aaronabraham311) [reply src/Components/Admin/EnvironmentEditor/environment.js:61]: Perhaps `editSceneInfo` would be better?

**Review by aaronabraham311 — APPROVED** [review]
Great work, Kevin! Hope this taught you a lot about React states, hooks and how to use data from the backend.

**Review by ahmedhamodi — APPROVED** [review]
Tested locally, looks good.
